weed/server/volume_server_handlers_read.go

```diff
@@ -147,15 +158,16 @@ func (vs *VolumeServer) GetOrHeadHandler(w http.ResponseWriter, r *http.Request)
 	if err != nil || count < 0 {
 		glog.V(3).Infof("read %s isNormalVolume %v error: %v", r.URL.Path, hasVolume, err)
 		if err == storage.ErrorNotFound || err == storage.ErrorDeleted {
-			w.WriteHeader(http.StatusNotFound)
+			NotFound(w)
 		} else {
-			w.WriteHeader(http.StatusInternalServerError)
+			InternalError(w)
 		}
 		return
 	}
 	if n.Cookie != cookie {
 		glog.V(0).Infof("request %s with cookie:%x expected:%x from %s agent %s", r.URL.Path, cookie, n.Cookie, r.RemoteAddr, r.UserAgent())
-		w.WriteHeader(http.StatusNotFound)
+		stats.VolumeServerRequestCounter.WithLabelValues(stats.ErrorGetNotFound).Inc()
```

Comment:
Look like twice increase metric counter

Reply:
Thanks
I fixed it here https://github.com/seaweedfs/seaweedfs/pull/4961